source/_data/themes.yml

```diff
@@ -1816,8 +1816,8 @@
     - google analytics
 - name: wind
   description: A light and pure theme for Hexo, in this complicated world.
-  link: https://github.com/Lonezj/hexo-theme-wind
-  preview: http://blog.mrdays.tech/
+  link: https://github.com/voezy/hexo-theme-wind
```

Comment:
The repo contains demo links to the old `preview` website.

Would you mind switching them to your proposed demo page?

Thank you :-)

Reply:
haha~ Ok then.

Reply:
It is ok now :-)